workspaces/github-actions/plugins/github-actions/src/components/WorkflowRunsCard/WorkflowRunsCard.tsx

```diff
@@ -314,9 +309,6 @@ export const WorkflowRunsCard = ({ entity }: WorkflowRunsCardProps) => {
     branch: branch === 'all' ? undefined : branch,
   });
 
-  const githubHost = hostname || 'github.com';
-  const hasNoRuns = !cardProps.loading && !runsData;
```

Comment:
This line is where the branching happens between the two empty states. When there are errors accessing GitHub, `runsData` is undefined, but when the request completes successfully but has no data, `runsData` is an empty array. This means that when there's an error, we end up rendering the content from line 340, but when there's no data, we render the content from line 355.

One way to test the error state is logging in with an OAuth app but not granting it access to the org containing the component(s) registered in the catalog.

Reply:
Is there any additional information we can surface about the error? It seems like `useWorkflowRuns` will post the error with the Error API but should the card also have the error?

Reply:
Possibly, but I'd suggest not - the GitHub actions plugin has a bunch of cards, so putting them all in an error state might be a little overwhelming.

Reply:
Ok, there's the Error API message that will show up so that should at least help point to a direction in these cases.